packages/patrol_cli/lib/src/ios/ios_test_backend.dart

```diff
@@ -13,6 +13,38 @@ import 'package:patrol_cli/src/devices.dart';
 import 'package:patrol_cli/src/ios/ios_deploy.dart';
 import 'package:process/process.dart';
 
+enum BuildMode {
+  debug('Debug'),
+  profile('Profile'),
+  release('Release');
+
+  const BuildMode(this.name);
```

Comment:
Do you ever intend on using this constructor? If not, make it private

Reply:
It's not needed anymore, removed. Thanks.

Reply:
> Do you ever intend on using this constructor? If not, make it private
It is an enum, you cannot construct a variant dynamically

Reply:
@shilangyu but you may use it for enum-value which is what I asked for